routers/web/repo/actions/actions.go

```diff
@@ -24,9 +28,10 @@ const (
 )
 
 type Workflow struct {
-	Entry     git.TreeEntry
-	IsInvalid bool
-	ErrMsg    string
+	Entry              git.TreeEntry
+	IsInvalid          bool
+	HaveMatchingRunner bool
+	ErrMsg             string
```

Comment:
How about combining them into `ErrMsg` and displaying the warning icon when it is not empty? Or we may need add more and more flags.

PS, I don't really think we need <img width="26" alt="image" src="https://user-images.githubusercontent.com/9418365/231969793-f5028398-6473-43ba-b33d-75bec5055cef.png"> when everything is fine, it's confusing, it looks like every job in the workflow resulted in success.